build.xml

```diff
@@ -375,6 +376,8 @@
     <include name="org/apache/catalina/startup/Tool.*" />
     <include name="org/apache/catalina/security/SecurityClassLoad.*" />
     <include name="org/apache/catalina/webresources/war/**" />
+    <include name="org/apache/coyote/Constants.*" />
```

Comment:
Your BZ doesn't show use of this, so where is this used ?

Reply:
it is transitively required by `Globals.java`